docs/experimental/node-split-poc.md

```diff
@@ -0,0 +1,154 @@
+# Node split PoC
+
+In our ongoing effort to lower the bar for smeshing, we're exploring a new direction for reducing system requirements for smeshers. The idea is to re-architect the internal modules in go-spacemesh, isolating the smeshing logic from the passive consensus code. By dividing the node along these lines, we can enable users to run the lightweight yet sensitive (requiring access to the private key) smeshing logic separately from the rest of the node. This allows users with limited resources to use a remote node for their smeshing.
+
+We demonstrate a PoC of the node split, where the smeshing logic is separated from the rest of the node. The PoC is based on the current go-spacemesh codebase and is intended to demonstrate the feasibility of the node split concept. The PoC is not intended to be a production-ready implementation.
+
+The PoC itself is still a work in progress so please expect changes and improvement in the future.
+
+The architecture of the PoC is as follows:
+
+![design.png](./../../static/img/node-split/design.png)
+
+The PoC consists of two separate processes:
+* `node service` aka node
+* `smesher service` aka client
+
+The easiest way to understand the setup is to try the PoC yourself. The following instructions will guide you through the process.
+
+
+## Why is this important?
+
+It's best to watch the [video](https://www.youtube.com/watch?v=d4jBz1krRHg) to get a better understanding of the problem and our reasoning behind the node split.
+
+The current go-spacemesh node is a monolithic application that includes all the logic required for consensus, smeshing, and other services. This makes it difficult to run the node on low-resource devices, as the node requires a significant amount of resources to run. By splitting the node into two separate services, we can enable users to run the smeshing logic on a separate device, while still connecting to a remote node for the rest of the services. This allows users to run the node on low-resource devices, while still participating in the Spacemesh network.
+
```

Comment:
It's actually already split to 4 (node, client, post-service, k2pow service). Perhaps you could add a note that each rounded rectangle can (but doesn't have to) represent a separate machine with different characteristics (optimized for different work).

Reply:
I will think about how to redo that part.

Reply:
Please check it now @poszu